src/renderer/themes/lens-light.json

```diff
@@ -34,7 +34,7 @@
     "buttonDisabledBackground": "#808080",
     "tableBgcStripe": "#f8f8f8",
     "tableBgcSelected": "#f4f5f5",
-    "tableHeaderBackground": "#f1f1f1",
+    "tableHeaderBackground": "#ffffff",
```

Comment:
It affects general tables style. It was better with "grayed" table head.

Tables before the change:
![gray table head](https://user-images.githubusercontent.com/9607060/120992289-216b6500-c78b-11eb-8ae9-e559286a113d.png)

Tables after the change:
![white table head](https://user-images.githubusercontent.com/9607060/120992463-4c55b900-c78b-11eb-9b51-d906190ba6d6.png)